mscore/inspector/inspector.cpp

```diff
@@ -1049,13 +1050,23 @@ void InspectorTremolo::setElement()
                   hasTabStaffTremolo = true;
                   break;
                   }
+            if (toTremolo(ee)->twoNotes()) {
```

Comment:
this is of course nonsense :-(

Reply:
actually the existing logic is flawed already: Beam style should be enabled if there is at least one minim two note Tremolo (and a nonTab-staff)
Will try to fix that first...

Reply:
I disagree. I think if there's at least one tremolo which is non-two-note-minim-tremolo, the beam style option should be disabled, because it doesn't make sense to other tremolos.

Reply:
If it makes sense for at least one of the selection, this needs to get shown.